test/integration/http_integration.cc

```diff
@@ -681,6 +681,25 @@ void HttpIntegrationTest::testRouterUpstreamProtocolError(const std::string& exp
   EXPECT_THAT(log, HasSubstr(expected_flag));
 }
 
+void HttpIntegrationTest::testDownstreamProtocolError() {
```

Comment:
Should you add a test in the ext_proc_integration_test.cc instead?  In there, you can easily create a test case with ext_proc filter configured, and send downstream request with BAD method, and check the response.

Reply:
Took me a while to figure out how to coax the listener to accept HTTP/1 connections properly, but the test has been moved there.